gnomad/utils/sparse_mt.py

```diff
@@ -319,11 +319,16 @@ def get_as_info_expr(
     if "AS_SB" in agg_expr:
         agg_expr["AS_SB_TABLE"] = agg_expr.pop("AS_SB")
 
+    if 'alt_alleles_range_array' not in mt.row or mt['alt_alleles_range_array'].dtype != hl.dtype('array<int32>'):
```

Comment:
@tpoterba Sorry if this is a stupid question, what is the motivation for having this instead of the same fix as in the `gnomad_qc` repo where you added.

```
mt = mt.annotate_rows(alt_alleles_range_array=hl.range(1, hl.len(mt.alleles)))
```
And then just drop the annotation after we use it rather than making it already be on the MT?